src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -74,178 +80,150 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="1"/>
-
     <features>
       <feature bit="0" code="SN" name="SceneNames" summary="The ability to store a name for a scene.">
         <optionalConform/>
       </feature>
     </features>
-    <attribute side="server" code="0x0000" name="LastConfiguredBy" define="LAST_CONFIGURED_BY" type="node_id" isNullable="true" optional="true">
-      <optionalConform/>
-    </attribute>
-    <attribute side="server" code="0x0001" name="SceneTableSize" define="SCENE_TABLE_SIZE" type="int16u" min="16" default="16">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0002" name="FabricSceneInfo" define="FABRIC_SCENE_INFO" type="array" entryType="SceneInfoStruct">
-      <mandatoryConform/>
-    </attribute>
-    
+    <attribute side="server" code="0x0000" name="SceneTableSize" define="SCENE_TABLE_SIZE" type="int16u" default="16">SceneTableSize</attribute>
+    <attribute side="server" code="0x0001" name="FabricSceneInfo" define="FABRIC_SCENE_INFO" type="array" entryType="SceneInfoStruct">FabricSceneInfo</attribute>
```

Comment:
This seems to be using an old version of alchemy, so it's adding the name of the attribute in multiple places?

Reply:
I pull the latest version and the generated output remains the same, when you say:

>  adding the name of the attribute in multiple places

You mean that `name="SceneTableSize"` should not be in the xml?

Reply:
I figured out the _ SceneTableSize_ at the end was the extra, removed.